traj/scripts/demo

```diff
@@ -66,7 +66,7 @@ max_jerks = np.deg2rad(np.array([
     8000.0,
     8000.0,
     16000.0,
-])) * 1.0
+]))
```

Comment:
These are not really hacks, are they? :)

The scaling factor is essentially a way to slow down the trajectory. A bit primitive, but quite OK for now I'd say.